Source/ALSV4_CPP/Public/Character/ALSCharacter.h

```diff
@@ -69,4 +85,101 @@ class ALSV4_CPP_API AALSCharacter : public AALSBaseCharacter
 
 	UPROPERTY(EditAnywhere, BlueprintReadWrite)
 	UStaticMeshComponent* StaticMesh = nullptr;
-};
+
+protected:
+	/** Colour System*/
+	UPROPERTY(EditAnywhere, BlueprintReadWrite, Category = "ALS|ColorSystem")
+	bool bSolidColor = true;
+
+	UPROPERTY(EditAnywhere, BlueprintReadWrite, Category = "ALS|ColorSystem")
+	bool bShoes = true;
+
+	UPROPERTY(EditAnywhere, BlueprintReadWrite, Category = "ALS|ColorSystem")
+	bool bGloves = true;
+
+	/** Default Colour System*/
+	UPROPERTY(EditAnywhere, BlueprintReadWrite, Category = "ALS|ColorSystem")
+	FLinearColor ALSDefaultColor = FLinearColor(0.2f, 0.2f, 1.0f, 1.0f);
```

Comment:
I'm not huge fan of bringing color system into C++ code. It's all unnecessary clutter in header file.

Reply:
Alright I'll just revert the changes for the color system 👍